code-challenges.test.js

```diff
@@ -49,9 +107,35 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
-
-// b) Create the function that makes the test pass.
-
+describe("upperCase", () => {
+    it("takes in an array of words and returns an array with all the words capitalized.", () => {
+      const randomNouns1 = []
+      const randomNouns2 = []
+      expect(upperCase(randomNouns1))
+      expect(upperCase(randomNouns2))
+    })
+  })
```

Comment:
Just like before, make sure to compare to an expected output.  Also, your test variables inside the test are set to an empty array.  I like the placement that you have for those test variables since they are within the scope you are working with, just be sure that you copy the test data into those variables as well.